base/tks/src/org/dogtagpki/server/tks/rest/TPSConnectorService.java

```diff
@@ -314,14 +315,14 @@ public Response createSharedSecret(String id) {
             }
 
 //            CryptoUtil.createSharedSecret(nickname);
-            CryptoUtil.createSharedSecret(nickname,KeyGenAlgorithm.AES,AES_SESS_KEYSIZE);
+            CryptoUtil.createSharedSecret(nickname,KeyGenAlgorithm.AES,AES_SESS_KEYSIZE_256);
```

Comment:
so the 4 changes (this and the 3 below) from AES_SESS_KEYSIZE  to  AES_SESS_KEYSIZE_256 won't cause any issue for older tokens?

Reply:
No, this is just for the shared secret between tks and tps which wraps blobs being sent back and forth. The token in use is not a factor here.